### PR TITLE
Improved documentation

### PR DESCRIPTION
**Pull request by pkubanek**

### CODE REVIEW

File: include/cRIO/ElectromechanicalPneumaticILC.h [hunk 1]
```diff
@@ -56,26 +68,50 @@ class ElectromechanicalPneumaticILC : public virtual ILC::ILCBusList {
...
+     * @ingroup M2
      */
     void reportStepperForceStatus(uint8_t address) { callFunction(address, 67, 1800); }
 
     /**
-     * Unicast command to read hardpoint LVDT. ILC command 122 (0x7a).
+     * Unicast command to set a single force actuator force offset.
      *
      * @param address ILC address
+     * @param slewFlag DCA booster valve
```
Comment (teweitsai) [inline include/cRIO/ElectromechanicalPneumaticILC.h:83]: What is DCA? Do you have the full name somewhere to explain this?

File: include/cRIO/ElectromechanicalPneumaticILC.h [hunk 2]
```diff
@@ -112,16 +161,35 @@ class ElectromechanicalPneumaticILC : public virtual ILC::ILCBusList {
...
      * Read ILC mezzanine pressure. ILC command code 119 (0x77).
      *
      * @param address ILC address
+     *
+     * @ingroup M1M3_hp
      */
     void reportMezzaninePressure(uint8_t address) { callFunction(address, 119, 1800); }
 
+    /**
+     * Unicast command to read hardpoint LVDT. ILC command 122 (0x7a).
```
Comment (teweitsai) [inline include/cRIO/ElectromechanicalPneumaticILC.h:183]: What is the LVDT? Full name?

File: include/cRIO/ElectromechanicalPneumaticILC.h [hunk 3]
```diff
@@ -150,10 +222,34 @@ class ElectromechanicalPneumaticILC : public virtual ILC::ILCBusList {
      * @param address returned from this ILC
      * @param breakawayLVDT breakway LVDT value
      * @param displacementLVDT displacement LVDT value
+     *
+     * @ingroup M1M3_hp
      */
     virtual void processHardpointLVDT(uint8_t address, float breakawayLVDT, float displacementLVDT) = 0;
 
+    /**
+     * Called when response from call to command 210 or 220 is received. Called
```
Comment (teweitsai) [inline include/cRIO/ElectromechanicalPneumaticILC.h:231]: What are the command 210 and 220? Maybe consider to use the enum for commands for more meaningful names.

File: include/cRIO/ElectromechanicalPneumaticILC.h [hunk 4]
```diff
@@ -167,11 +263,26 @@ class ElectromechanicalPneumaticILC : public virtual ILC::ILCBusList {
...
+     * @ingroup M1M3_fa
+     * @ingroup M1M3_hp
+     * @ingroup M2
      */
     virtual void processCalibrationData(uint8_t address, float mainADCK[4], float mainOffset[4],
                                         float mainSensitivity[4], float backupADCK[4], float backupOffset[4],
                                         float backupSensitivity[4]) = 0;
 
+    /**
+     * Called when response from call to command 119 is read.
```
Comment (teweitsai) [inline include/cRIO/ElectromechanicalPneumaticILC.h:276]: Yep, what is 119? Please consider to use the enum for these commands for more meaningful names.

**Review by teweitsai — APPROVED** [review]
Looks good.

File: include/cRIO/ElectromechanicalPneumaticILC.h (hunk 2 above)
Comment (pkubanek) [reply include/cRIO/ElectromechanicalPneumaticILC.h]: Glossary added.

File: include/cRIO/ElectromechanicalPneumaticILC.h (hunk 1 above)
Comment (pkubanek) [reply include/cRIO/ElectromechanicalPneumaticILC.h]: Glossary added.

File: include/cRIO/ElectromechanicalPneumaticILC.h (hunk 4 above)
Comment (pkubanek) [reply include/cRIO/ElectromechanicalPneumaticILC.h:295]: Changed to enums, added pointers to LTS

File: include/cRIO/ElectromechanicalPneumaticILC.h (hunk 3 above)
Comment (pkubanek) [reply include/cRIO/ElectromechanicalPneumaticILC.h:250]: Changed to enums, added pointers to LTS

**Pull request merged**